src/contribution/projects-explore.js

```diff
@@ -11,23 +11,46 @@ window.c.contribution.ProjectsExplore = ((m, c, h, _) => {
 
         controller: () => {
             const filters = m.postgrest.filtersVM,
+                  follow = c.models.categoryFollower,
                   byCategory = filters({state_order: 'gte', category_id: 'eq'}).state_order('published'),
                   projects = m.postgrest.paginationVM(c.models.project, 'project_id.desc'),
-                  lCategories = m.postgrest.loaderWithToken(
-                      c.models.category.getPageOptions(
-                          filters({}).order({name: 'asc'}).parameters()
-                      )
-                  ),
                   filtersMap = c.contribution.projectFilters(),
                   categoryCollection = m.prop([]),
                   title = m.prop(),
-                  category = m.prop(),
+                  categoryId = m.prop(),
+                  findCategory = (id) => {
+                      return _.find(categoryCollection(), function(c){ return c.id === parseInt(id); });
+                  },
+                  category = _.compose(findCategory, categoryId),
+
+                  loadCategories = () => {
+                      return c.models.category.getPageWithToken(filters({}).order({name: 'asc'}).parameters()).then(categoryCollection);
+                  },
+
+                  reloadCategories = () => {
```

Comment:
As we are not using loaders here, setting background to false on the request options should make this function unnecessary. Think it's worth it?